src/pizzeria/Order.java

```diff
@@ -0,0 +1,84 @@
+package pizzeria;
+
+import java.time.LocalTime;
+import java.util.*;
+
+public class Order {
+    private final int orderNumber;
+    private final int customerNumber;
+    private final List<String> pizzaName;
+    private final String pizzaType;
+    private final List<Integer> quantityOfOrder;
+    private final LocalTime time = LocalTime.now();
+    private final Set<Ingredients> getFilledPizza = new HashSet<>();
+
+    Order(int customerNumber, List<String> pizzaName, List<Integer> quantityOfOrder, String pizzaType) {
+        Random order = new Random();
+        this.orderNumber = order.nextInt(100000);
```

Comment:
Use [THIS URL](https://stackoverflow.com/questions/7815196/how-to-generate-a-random-five-digit-number-java) to create random 5 digits number.

Reply:
done